spring-integration-core/src/main/java/org/springframework/integration/config/annotation/AbstractMethodAnnotationPostProcessor.java

```diff
@@ -72,6 +70,8 @@
 import org.springframework.util.ObjectUtils;
 import org.springframework.util.StringUtils;
 
+import org.aopalliance.aop.Advice;
+
 /**
```

Comment:
Maybe I need to review my settings - my IDE moves it back - I need to check against the SF wiki doc.